lib/plugins/aws/provider/awsProvider.test.js

```diff
@@ -51,6 +53,73 @@ describe('AwsProvider', () => {
       delete process.env.proxy;
     });
 
+    it('should set AWS ca single', () => {
+      process.env.ca = '-----BEGIN CERTIFICATE-----\n-----END CERTIFICATE-----';
+      const newAwsProvider = new AwsProvider(serverless, options);
+
+      expect(typeof newAwsProvider.sdk.config.httpOptions.agent).to.not.equal('undefined');
+
+      // clear env
+      delete process.env.ca;
+    });
+
+    it('should set AWS ca multiple', () => {
+      const certContents = '-----BEGIN CERTIFICATE-----\n-----END CERTIFICATE-----';
+      process.env.ca = `${certContents},${certContents}`;
+      const newAwsProvider = new AwsProvider(serverless, options);
+
+      expect(typeof newAwsProvider.sdk.config.httpOptions.agent).to.not.equal('undefined');
+
+      // clear env
+      delete process.env.ca;
+    });
+
+    it('should set AWS cafile single', () => {
+      const tmpdir = os.tmpdir();
+      const certContents = '-----BEGIN CERTIFICATE-----\n-----END CERTIFICATE-----';
+      fs.writeFileSync(`${tmpdir}/ca.txt`, certContents);
+      process.env.cafile = `${tmpdir}/ca.txt`;
+      const newAwsProvider = new AwsProvider(serverless, options);
+
+      expect(typeof newAwsProvider.sdk.config.httpOptions.agent).to.not.equal('undefined');
+
+      // clear env
+      fs.unlink(`${tmpdir}/ca.txt`);
+      delete process.env.cafile;
+    });
+
+    it('should set AWS cafile multiple', () => {
+      const tmpdir = os.tmpdir();
+      const certContents = '-----BEGIN CERTIFICATE-----\n-----END CERTIFICATE-----';
+      fs.writeFileSync(`${tmpdir}/ca1.txt`, certContents);
+      fs.writeFileSync(`${tmpdir}/ca2.txt`, certContents);
+      process.env.cafile = `${tmpdir}/ca1.txt,${tmpdir}/ca2.txt`;
+      const newAwsProvider = new AwsProvider(serverless, options);
+
+      expect(typeof newAwsProvider.sdk.config.httpOptions.agent).to.not.equal('undefined');
+
+      // clear env
+      fs.unlink(`${tmpdir}/ca1.txt`);
+      fs.unlink(`${tmpdir}/ca2.txt`);
+      delete process.env.cafile;
+    });
+
+    it('should set AWS ca and cafile', () => {
+      const tmpdir = os.tmpdir();
+      const certContents = '-----BEGIN CERTIFICATE-----\n-----END CERTIFICATE-----';
+      fs.writeFileSync(`${tmpdir}/ca.txt`, certContents);
+      process.env.ca = certContents;
+      process.env.cafile = `${tmpdir}/ca.txt`;
```

Comment:
Could use `path.join` here to make this OS independent?

Reply:
complete.